nixos/modules/services/x11/window-managers/awesome.nix

```diff
@@ -11,6 +11,9 @@ let
     " --search " + (getLuaPath path "share") +
     " --search " + (getLuaPath path "lib")
   );
+  awesomeWrapped = (writeScriptBin ".awesome-wrapped" ''
```

Comment:
Normally with `wrapProgram` the wrapper becomes available under the normal name and the original executable is moved to `.$exeName-wrapped` - the opposite of what you're doing here.

I don't see a reason to keep the non-wrapped `awesome` around, so to avoid confusion I think it makes sense to use plain `awesome` as the name for the wrapper and remove the original `awesome` from `systemPackages`.

If there's a good reason to keep the original around, you could use a name like `awesome-wrapped` or something similar.

Reply:
The Idea was to not break anything. On closer inspection, I wouldn't know what could break though. Your reasoning sounds perfectly reasonable to me, so I will adjust accordingly.